examples/smb2-share-enum.c

```diff
@@ -80,6 +83,20 @@ void se_cb(struct smb2_context *smb2, int status,
 
 int main(int argc, char *argv[])
 {
+	WORD wVersionRequested;
```

Comment:
This looks good but you will need to wrap this inside an ifdef so that it is only enabled when compiling on windows.